camera-viewfinder-compose/build.gradle.kts

```diff
@@ -0,0 +1,71 @@
+plugins {
+    id("com.android.library")
+    id("org.jetbrains.kotlin.android")
+}
+
+android {
+    namespace = "com.google.jetpackcamera.camerax"
+    compileSdk = 33
+
+    defaultConfig {
+        minSdk = 21
+        targetSdk = 33
```

Comment:
I'm assuming these will reference vars in a follow up PR?

Reply:
Yup, sorry should've added that to the description.

Follow up PRs for

- project-wide values
- Moving to version catalogs
- Potentially using convention plugins: https://docs.gradle.org/current/samples/sample_convention_plugins.html